usage_examples/README.md

```diff
@@ -100,7 +100,7 @@ in the `spacy_llm/tasks/` directory. We also recommend checking out the
 ## Using LangChain and other integrated third-party prompting libraries
 
 `spacy-llm` integrates bindings to a number of libraries centered on prompt management and LLM usage to allow users
```

Comment:
I would still update this bit "integrates a number of libraries" as we really only have one, now.

Reply:
Done.